wal/src/config.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 The CeresDB Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    message_queue_impl::config::KafkaWalStorageConfig, rocks_impl::config::RocksDBWalStorageConfig,
+    table_kv_impl::config::ObkvWalStorageConfig,
+};
+
+/// Options for wal storage backend
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(tag = "type")]
+pub enum WalStorageConfig {
```

Comment:
This is enum is already in wal module, I suggest we name it `StorageConfig` directly.